### ISSUE TITLE
Broken ARIA reference + aria-label Object

### ISSUE THREAD

**Issue by 5ulo** [somewebmedia/hc-offcanvas-nav#55]
The `aria-labelledby="hc-nav-1"` doesn't match anything in the DOM. This id should be attached as `<h2 class="nav-title" id="hc-nav-1">`

And the next problem is with page logo as a offcanvas title.. as it is object, it also renders as object inside `aria-label` and looks like `aria-label="[object HTMLImageElement]"`

Image says it all..

![Screen Shot 2020-11-30 at 09 58 03](https://user-images.githubusercontent.com/4980919/100589407-5b970e00-32f3-11eb-968f-2b49d554e888.png)

edit: `aria-labelledby` is broken in all submenus too

**Comment by somewebmedia** [comment]
Yeah I see now I messed up with the aria. It'll be fixed in the next release.

**Comment by somewebmedia** [comment]
It's properly (hope so) fixed in the latest release.
